README.Rmd

```diff
@@ -60,8 +58,8 @@ To be included as an `{admiral}` package extension we expect developers to adher
 
 * `r dyn_link("Code of Conduct", admiral_homepage, "CODE_OF_CONDUCT.html")`
 * `r dyn_link("Manifesto", admiral_homepage, "index.html#admiral-manifesto")`
```

Comment:
I think this should link to `index.html` in admiral (not to `index.html` in the extension package).

I am not sure about `CODE_OF_CONDUCT.html`. Should we have only one in admiral or in each extension package?

Reply:
that's what it does with admiral_homepage here, e.g. bottom one links to https://pharmaverse.github.io/admiral/index.html#admiral-manifesto. Only one code of conduct should be sufficient across admiral family